packages/dds/map/src/test/mocha/browser.spec.ts

```diff
@@ -0,0 +1,78 @@
+/*!
```

Comment:
Ideally, we could find an off-the-shelf solution to what this module is doing. This is why I originally mentioned `karma` as an option. It handles this sort of thing with minimal configuration. I suspect there are other options out there.